app.js

```diff
@@ -0,0 +1,17 @@
+const express = require('express')
+const app = express()
+const getTopics = require('./controller')
+
+app.get('/api/topics', getTopics)
+
+
+app.use((err, req, res, next) => {
```

Comment:
this isn't quite what you want to do here. What you have is a bit of error handling middleware that will need to be passed an error from your controller. In the case of an invalid route, there will be no error there's just nothing happening, so you would want to set any other route other than the ones specified above to send whatever status and message. Have a look at this link https://www.geeksforgeeks.org/how-to-redirect-404-errors-to-a-page-in-express-js/ for how you can route all other routes to send back a 404 if they don't exist